README.md

```diff
@@ -24,11 +24,11 @@ $sam->commandSAM( "HELLO VERSION MIN=3.0 MAX=3.1 \n" );
 ```
 
 You can also use composer for this.
-```
+```php
```

Comment:
This is a shell command :)

Reply:
https://til.hashrocket.com/posts/szgdi4at59-github-insert-a-suggestion